src/pages/Components/Notification/previews/notification-states.preview.scss

```diff
@@ -0,0 +1,18 @@
+@import 'settings-tools/_all-settings';
+@import 'components/_c.notification';
+
+body {
+  padding-bottom: 100px;
+}
+
```

Comment:
why do you change the body ?

Reply:
I was trying to extend the frame height in order to not hide the content.